woocommerce-yotpo/wc_yotpo.php

```diff
@@ -72,7 +72,8 @@ function wc_yotpo_front_end_init() {
 		if($settings['bottom_line_enabled_product']) {	
 			add_action('woocommerce_single_product_summary', 'wc_yotpo_show_buttomline',7);	
 			wp_enqueue_style('yotpoSideBootomLineStylesheet', plugins_url('assets/css/bottom-line.css', __FILE__));
-		}			
+		}
+		add_action('woocommerce_before_single_product', 'wc_yotpo_show_rs', 5);		
```

Comment:
@hxii this will be reflected in the settings page, or enabled by default?

Reply:
@billykov enabled by default

Reply:
How can you disable it? only by editing the code?

Reply:
@billykov correct. Settings switch can be added though.

Reply:
@billykov checkbox was added under the admin page. It is disabled by default.